src/index.ts

```diff
@@ -258,9 +259,9 @@ function init(modules: { typescript: typeof ts_module }) {
                   description: `Disable rule '${problem.getRuleName()}'`,
                   changes: [{
                     fileName: fileName,
-                    textChanges: [{

```

Comment:
hmm, I didn't intend to edit these lines at all and I cannot spot any edits. Could be a change the line terminator. I would have to redo the PR to fix this. Let me know if you want me to do this.